azure_jumpstart_ag/artifacts/PowerShell/Bootstrap.ps1

```diff
@@ -116,115 +178,149 @@ Invoke-WebRequest ($templateBaseUrl + "artifacts/icons/contoso.svg") -OutFile $A
 
 New-Item -path alias:kubectl -value 'C:\ProgramData\chocolatey\lib\kubernetes-cli\tools\kubernetes\client\bin\kubectl.exe'
 
+##############################################################
 # Disable Network Profile prompt
+##############################################################
 $RegistryPath = "HKLM:\System\CurrentControlSet\Control\Network\NewNetworkWindowOff"
 if (-not (Test-Path $RegistryPath)) {
-    New-Item -Path $RegistryPath -Force | Out-Null
+  New-Item -Path $RegistryPath -Force | Out-Null
 }
 
+##############################################################
+# Updating Microsoft Edge startup settings
+##############################################################
 # Disable Microsoft Edge sidebar
-$RegistryPath = 'HKLM:\SOFTWARE\Policies\Microsoft\Edge'
-$Name         = 'HubsSidebarEnabled'
-$Value        = '00000000'
+$Name = 'HubsSidebarEnabled'
 # Create the key if it does not exist
-If (-NOT (Test-Path $RegistryPath)) {
-  New-Item -Path $RegistryPath -Force | Out-Null
+If (-NOT (Test-Path $AgConfig.EdgeSettingRegistryPath)) {
+  New-Item -Path $AgConfig.EdgeSettingRegistryPath -Force | Out-Null
 }
-New-ItemProperty -Path $RegistryPath -Name $Name -Value $Value -PropertyType DWORD -Force
+New-ItemProperty -Path $AgConfig.EdgeSettingRegistryPath -Name $Name -Value $AgConfig.EdgeSettingValueFalse -PropertyType DWORD -Force
 
 # Disable Microsoft Edge first-run Welcome screen
-$RegistryPath = 'HKLM:\SOFTWARE\Policies\Microsoft\Edge'
-$Name         = 'HideFirstRunExperience'
-$Value        = '00000001'
+$Name = 'HideFirstRunExperience'
 # Create the key if it does not exist
-If (-NOT (Test-Path $RegistryPath)) {
-  New-Item -Path $RegistryPath -Force | Out-Null
+If (-NOT (Test-Path $AgConfig.EdgeSettingRegistryPath)) {
+  New-Item -Path $AgConfig.EdgeSettingRegistryPath -Force | Out-Null
 }
-New-ItemProperty -Path $RegistryPath -Name $Name -Value $Value -PropertyType DWORD -Force
+New-ItemProperty -Path $AgConfig.EdgeSettingRegistryPath -Name $Name -Value $AgConfig.EdgeSettingValueTrue -PropertyType DWORD -Force
 
 # Disable Microsoft Edge "Personalize your web experience" prompt 
-$RegistryPath = 'HKLM:\SOFTWARE\Policies\Microsoft\Edge'
-$Name         = 'PersonalizationReportingEnabled'
-$Value        = '00000000'
+$Name = 'PersonalizationReportingEnabled'
 # Create the key if it does not exist
-If (-NOT (Test-Path $RegistryPath)) {
-  New-Item -Path $RegistryPath -Force | Out-Null
+If (-NOT (Test-Path $AgConfig.EdgeSettingRegistryPath)) {
+  New-Item -Path $AgConfig.EdgeSettingRegistryPath -Force | Out-Null
 }
-New-ItemProperty -Path $RegistryPath -Name $Name -Value $Value -PropertyType DWORD -Force
+New-ItemProperty -Path $AgConfig.EdgeSettingRegistryPath -Name $Name -Value $AgConfig.EdgeSettingValueFalse -PropertyType DWORD -Force
 
+##############################################################
 # Installing Posh-SSH PowerShell Module
+##############################################################
```

Comment:
thinking about it, we dont really need PoSH-SSH in Agora, it doesnt hurt though. Just a thought.